packages/widgetbook_core/test/src/navigation_tree/bloc/navigation_bloc_test.dart

```diff
@@ -131,6 +136,23 @@ void main() {
       ],
     );
 
+    blocTest<NavigationBloc, NavigationState>(
+      'Emits state with filtered nodes when $LoadNavigationTree event is added '
+      'on an existing query',
+      build: NavigationBloc.new,
+      act: (bloc) => bloc.add(
```

Comment:
@Roaa94 I matched your style here, but I think this should be named "emits state ..." with a lowercase first letter. 

This is because the test cases are printed like
```
✓ NavigationBloc Emits state with correct nodes when LoadNavigationTree event is added
✓ NavigationBloc Emits state with filtered nodes when LoadNavigationTree event is added on an existing query
```

Just a little FYI 😀